submissions/UA-girl/Task-HTML-CSS/index.html

```diff
@@ -1466,11 +1843,11 @@ <h2 class="section-header order-header">Ваше замовлення</h2>
           <table class="table order-table">
             <tr>
               <th class="table-header">Дата:</th>
-              <td class="table-data">30.01.2012p.</td>
+              <td class="table-data"><time>30.01.2012p.</time></td>
             </tr>
             <tr>
               <th class="table-header">Час:</th>
-              <td class="table-data">13:00</td>
+              <td class="table-data"><time>13:00</time></td>
```

Comment:
Same story as in line 1846.

Reply:
Fixed this and removed figure. Thank you very much for advice!